### ISSUE TITLE
Mistakes in `aachen/syn/sqr/spec.yml`

### ISSUE THREAD

**Issue by delcypher** [danielschemmel/fp-benchmarks-aachen#11]
The spec for this file and the corresponding source file have some problems.

For the `aachen/syn/sqr_double-flow.x86_64.bc` benchmark we observe

```
KLEE: ERROR: /home/user/fp-bench/benchmarks/c/aachen/syn/sqr/test.c:45: ASSERTION FAIL: a * 0.99 <= b && a >= b * 0.99 && "this should have an error path..."
```

According to the spec we should only see an error on line 47.

Taking a closer look at ``aachen/syn/sqr/spec.yml` it suggests to me that.

* For `double-flow` the `ENABLE_FLOW` define should be called `ENABLE_INFINITY` in the spec
* For `longdouble-flow` the `ENABLE_FLOW` define should be called `ENABLE_INFINITY` in the spec

in order for the provided counter examples to make sense. Of course you could rename the macro in the source file instead. I don't mind which.

**Comment by delcypher** [comment]
@danielschemmel Ping. Please fix these benchmarks.

**Comment by danielschemmel** [comment]
should be fixed as of aafa215b6d8598536cc3777405931459f0268093

**Comment by delcypher** [comment]
@danielschemmel That commit doesn't seem to be in your master branch. Have you pushed it yet?

**Comment by danielschemmel** [comment]
@delcypher Sorry, I already noticed that I pushed the wrong repo, it should be fixed as of da0d21e1e1f817e98cbabdace30e53cedbedfaa5 in this repo.
